Formula/servus.rb

```diff
@@ -0,0 +1,30 @@
+class Servus < Formula
+  desc "Library and Utilities for zeroconf networking"
+  homepage "https://github.com/HBPVIS/Servus"
+  url "https://github.com/HBPVIS/Servus/archive/1.5.0.tar.gz"
+  sha256 "42fb9c060f17f040ad3c7563f5e87c89f5a221a5aa7da21384fc26b9c725ecc8"
+
+  bottle do
+    cellar :any
+    sha256 "fb13c90a02d860b5442a046a66a98f190788b2d5f576c574b891751a14353893" => :el_capitan
+  end
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+
+    # Install tests to share/Servus/tests so that we can test later
```

Comment:
This formula has nothing in `bin` to test with (and you wouldn't want to put these executables into `bin` because they have very generic names like `uri` that might conflict with other tools).  Would you prefer that I take the source code of these tests, embed it into the formula, compile it at test time and test with that, kind of like how [libxml2 does it](https://github.com/Homebrew/homebrew-core/blob/master/Formula/libxml2.rb#L54-L71)?